bin/manage_addon.sh

```diff
@@ -0,0 +1,113 @@
+#!/bin/bash
+# Copyright 2015 Solinea, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+export DJANGO_SETTINGS_MODULE=goldstone.settings.docker_dev
+DOCKER_VM="default"
+APP_CONTAINER=goldstoneserver_gsappdev_1
+ADDON_NAME=""
+ADDON_FILE=""
+MODULE_NAME=""
+OPERATION="install"
+
+TOP_DIR=${GS_PROJ_TOP_DIR:-~/devel/goldstone-server}
+
+function usage() {
+    echo "Usage: $0 [--install] --addon-name=name --addon-file=filename [--docker-vm=name] [--app-container=name] |"
+    echo "          --uninstall --addon-name=name --package-name=py-package [--docker-vm=name] [--app-container=name]"
+}
+
+
+for arg in "$@" ; do
+    case $arg in
+        --install)
+            OPERATION=install
+        ;;
+        --uninstall)
+            OPERATION=uninstall
+        ;;
+        --docker-vm=*)
+            DOCKER_VM="${arg#*=}"
+            shift
+        ;;
+        --app-container=*)
+            APP_CONTAINER="${arg#*=}"
+            shift
+        ;;
+        --addon-name=*)
+            ADDON_NAME="${arg#*=}"
+            shift
+        ;;
+        --package-name=*)
+            MODULE_NAME="${arg#*=}"
+            shift
+        ;;
+        --addon-file=*)
+            ADDON_FILE="${arg#*=}"
+            shift
+        ;;
+        --help)
+            usage
+            exit 0
+        ;;
+        *)
+            # unknown option
+            usage
+            exit 1
+        ;;
+    esac
+done
+
+cd $TOP_DIR || exit 1
+
+if [[ ${ADDON_NAME} == "" ]] ; then
+    usage
+    exit 1
+fi
+
+  
+docker-machine start ${DOCKER_VM}
+eval "$(docker-machine env ${DOCKER_VM})"
+
+if [[ $(docker ps -f name=goldstoneserver_gsappdev_1 --format "{{.ID}}" | sed -n '$=') != "1" ]] ; then
+    echo "Container ${APP_CONTAINER} must be running.  Did you run start_dev_env.sh first?"
+    exit 1
+fi
+
+if [[ ${OPERATION} == "install" ]] ; then
```

Comment:
I got this error when I tried installing an add-on:

<img width="1124" alt="screen shot 2015-09-24 at 2 50 45 pm" src="https://cloud.githubusercontent.com/assets/231148/10087940/01a907c4-62cc-11e5-966d-e1fea9129b9d.png">

Poking around: In the container, the leases app was installed in /home/app.  The error message comes from addon_fabfile.py line 304.  The codetried to import the leases package and raised an ImportError Exception.  This is Python's way of saying, "file not found."